mods/screwdriver/init.lua

```diff
@@ -150,7 +150,9 @@ minetest.register_tool("screwdriver:screwdriver", {
 		return itemstack
 	end,
 	on_place = function(itemstack, user, pointed_thing)
-		screwdriver.handler(itemstack, user, pointed_thing, screwdriver.ROTATE_AXIS, 200)
+		if user then
```

Comment:
screwdriver.handler should handle nil users

Reply:
can user be nil in ndef.on_rotate?